src/org/labkey/targetedms/TargetedMSModule.java

```diff
@@ -371,6 +374,30 @@ public WebPartView getWebPartView(@NotNull ViewContext portalCtx, @NotNull Porta
             }
         };
 
+        BaseWebPartFactory passportFactory = new BaseWebPartFactory("Passport")
+        {
+            @Override
+            public WebPartView getWebPartView(@NotNull ViewContext portalCtx, @NotNull Portal.WebPart webPart)
+            {
+                QueryView v =  ProteinListView.createView(portalCtx);
+                v.setTitle("Passport");
+                v.setFrame(WebPartView.FrameType.PORTAL);
+                return v;
+            }
+        };
+
+        BaseWebPartFactory passportFactory = new BaseWebPartFactory("Passport")
```

Comment:
Does this compile? Looks like a duplicate declaration, and TeamCity is failing on the targetedms module compilation.